hasura/migrations/default/1703175768541_create_roles_table/up.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE "public"."roles" (
+  "id" varchar(50) NOT NULL DEFAULT gen_random_friendly_id ('rol'),
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "updated_at" timestamptz NOT NULL DEFAULT now(),
+  "user_id" varchar(50) NOT NULL,
+  "role" text NOT NULL,
```

Comment:
probably varchar is enough

Reply:
@pdtfh I've created it as `text` to match enum table `value` field type. In the [docs](https://hasura.io/docs/latest/schema/postgres/enums/#pg-create-enum-table) it have text type. Not sure if it will work with varchar.